src/components/svg/wrap/svg-keyboard-wrap.component.ts

```diff
@@ -194,13 +202,33 @@ export class SvgKeyboardWrapComponent implements OnInit, OnChanges {
             posTop = position.top + position.height;
         }
 
-        let content: {
-            name: string,
-            value: string
-        }[] = [];
+        this.tooltipData = {
+            posLeft: posLeft,
+            posTop: posTop,
+            content: this.getKeyActionContent(keyAction),
+            show: true
+        };
+    }
+
+    hideTooltip() {
+        this.tooltipData.show = false;
+    }
+
+    hidePopover(): void {
+        this.popoverShown = false;
+    }
 
+    selectLayer(index: number): void {
+        this.currentLayer = index;
+    }
+
+    private getKeyActionContent(keyAction: KeyAction): typeof SvgKeyboardWrapComponent.prototype.tooltipData.content {
```

Comment:
I would create interface for it or paste definition from the top `Observable<{ name: string, value: string }[]>`